readme.md

```diff
@@ -16,7 +16,12 @@ the square brackets (for multiple-choice questions).
 Using jQuery, write code that makes an AJAX get request to "http://kittengifs.com/gifs". Output to the console the total number of results you get back. (Assume the server responds with a JSON array of objects representing gifs.)
 
 Your Answer:
-```js
+```$.ajax({
+  url: http://kittengifs.com/gifs,
+  data: String,
+  success: null,
+  dataType: String
+});
 
```

Comment:
The broad structure is correct, but the values for dataType and data was way off.